### ISSUE TITLE
Source categories and criteria

### ISSUE THREAD

**Issue by joel-huang** [OpenSUTD/awesome-sutd#3]
We'll need some kind of filtering for submitted sources. Some initial ideas:

1. Have multiple awesome lists for different classes of sources, e.g. `freshmore`, `capstone`, `urop`, etc.
2. Projects should be at least a complete prototype, i.e. working
3. Encourage projects to submit an image of fixed dimensions, so we can spin off a landing page for projects in a grid/card view:

![image](https://user-images.githubusercontent.com/25588253/46775288-32911900-cd39-11e8-8381-ffd66d82dd5c.png)

**Comment by randName** [comment]
one thing is to cater for Github project pages being a good way to split stuff up, i.e. a repo called `capstone` will have the url `opensutd.github.io/capstone`, and each project repo can have it's own `gh-pages` branch that will be the project site

**Comment by tlkh** [comment]
If you think about it, awesome-list and the "official webpage" i.e. `opensutd.github.io` does serve the same purpose (act as a sorta more organised front end) We could do it that way. 

It might be more seamless to fold everything into one repo/github site (except for the individual project sites of course). Naturally that kinda means we can close up this repo.. haha

**Comment by joel-huang** [comment]
> If you think about it, awesome-list and the "official webpage" i.e. `opensutd.github.io` does serve the same purpose (act as a sorta more organised front end) We could do it that way.
> 
> It might be more seamless to fold everything into one repo/github site (except for the individual project sites of course). Naturally that kinda means we can close up this repo.. haha

I figure that would work **iff** the front end wholly replaces the awesome list. Using them as GH repositories then becomes a huge hassle if they're in `opensutd.github.io` together with the webapp src, etc.

**Comment by tlkh** [comment]
As an intermediate thing, I'm wondering if something like this makes sense:

[https://opensutd.github.io/projects.html](https://opensutd.github.io/projects.html)

**Comment by joel-huang** [comment]
Yes! Except the list would be kept separately, pristine, and still accessible, and the webapp pulls from the list programatically?

**Comment by tlkh** [comment]
I think we can do it such that the list is maintained in such a way a webapp can just parse it. Sorta like a custom markdown parser.

**Comment by joel-huang** [comment]
Additionally, I think a visual experience is fairly necessary + parallel to the overall vision of making work visible. We could encourage coupling images to each project (or thumbnails at least). I'm sure we can find better templates, but for an example: https://startbootstrap.com/template-overviews/4-col-portfolio/
